server/__init__.py

```diff
@@ -18,6 +18,7 @@
 import server.pcr
 import server.athletics
 import server.nso
+import server.studyspaces
```

Comment:
All scraping code should be in the Penn SDK library so that it's reusable across projects